packages/showcases/css/stories/components/breadcrumb/examples/overview.html

```diff
@@ -0,0 +1,50 @@
+<div class="block">
+  <nav aria-label="Breadcrumb" class="vtmn-breadcrumb">
+    <ol>
+      <li><a href="">Home</a></li>
+      <li><a href="">Page 2</a></li>
+      <li><a href="" aria-current="page">Current</a></li>
+    </ol>
+  </nav>
+</div>
+
+<div class="block">
+  <nav aria-label="Breadcrumb" class="vtmn-breadcrumb">
+    <ol>
+      <li>
+        <span class="vtmx-home-line" role="presentation"></span
```

Comment:
For icons I think it's better to use aria-hidden="true" like we can see here "for icons": https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute and because there's already a label so I think we don't need to specify a presentational element for screen readers.

Reply:
Yes you're right, aria-hidden is better in this case.